bin/vaultclient

```diff
@@ -98,28 +100,13 @@ program
     .command('create-account')
     .option('--name <NAME>')
     .option('--email <EMAIL>')
-    .option('--password <PASSWORD>')
     .action(action.bind(null, 'create-account', (client, args) => {
         client.createAccount(
-            args.name, { email: args.email, password: args.password },
+            args.name, { email: args.email },
             handleVaultResponse);
     }));
 
 program
-    .command('list-account-users')
```

Comment:
I don't remove it :) i have keep the list-accounts and remove account-users and access-keys :)